apps/yapms/src/lib/assets/maps/usa-senate-2024.svg

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg candidates='[{"id":"0", "name":"Democrats", "margins": [{ "color": "#1C408C" }, { "color": "#577CCC" }, { "color": "#8AAFFF" }, { "color": "#949BB3" }]},    {"id":"1", "name":"Republicans", "margins": [{ "color": "#BF1D29" }, { "color": "#FF5865" }, { "color": "#FF8B98" }, { "color": "#CF8980" }]}]' version="1.1" viewBox="0 0 959 593" xmlns="http://www.w3.org/2000/svg">
+<svg candidate-counts='[{"0":28},{"1":38}]' candidates='[{"id":"0", "name":"Democrats", "margins": [{ "color": "#1C408C" }, { "color": "#577CCC" }, { "color": "#8AAFFF" }, { "color": "#949BB3" }]},    {"id":"1", "name":"Republicans", "margins": [{ "color": "#BF1D29" }, { "color": "#FF5865" }, { "color": "#FF8B98" }, { "color": "#CF8980" }]}]' version="1.1" viewBox="0 0 959 593" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Would it be possible to instead have the starting count put inside the candidates property?

```[{"id": "0", "name":"Democrats", "count":"38"}]```

Reply:
Yep! Just pushed this. Less complex than I thought.

I changed the following
- Removed DefaultCountsSchema and DefaultCountsStoreSchema as all checking is now done w/ CandidateSchema
- Added an optional counts number argument to CandidateSchema
- Changed the Senate map to follow new formatting
- Added a null check for default-mode since it kept throwing errors